### PR TITLE
Get GitHub repos for org name

### PR DESCRIPTION
**Pull request by bafulton**
It required a little bit of refactoring, but I think it's for the better. Here are the major changes:
- Renamed the weblist loaders to "webpage" loaders, and extended github's webpage loader's functionality to load projects from org names.
- Reconfigured the "load" command. It now operates as follows: "load [registry] [type] name_or_file", eg: "load pypi list top5kmonth" or "load github org returntocorp".

### CODE REVIEW

**Comment by bafulton** [comment]
Note that this is a breaking change, since it reconfigures the load command.

File: r2c_isg/structures/dataset.py [hunk 1]
```diff
@@ -205,7 +226,7 @@ def extract_vars(obj: object) -> dict:
                 elif attr in ['uuids_', 'meta_']:
                     # convert lambda functions to strings
                     vars_dict[attr] = {
-                        key: getsource(func).split(': ', 1)[1].strip()
+                        key: getsource(func).split(': ', 1)[1].strip(',\n')
```
Comment (bafulton) [inline r2c_isg/structures/dataset.py]: This is an unrelated minor bugfix.

File: r2c_isg/apis/_api.py [hunk 2]
```diff
@@ -19,9 +19,9 @@ def __init__(self, **kwargs):
         self.nocache = False
 
         # call update to add any user-modifiable values
-        self.update(**kwargs)
+        self.configure(**kwargs)
 
-    def update(self, **kwargs):
+    def configure(self, **kwargs):
```
Comment (bafulton) [inline r2c_isg/apis/_api.py]: This is a much more intuitive name.

File: r2c_isg/apis/pypi.py [hunk 3]
```diff
@@ -49,7 +49,8 @@ def get_project(self, project: PypiProject, **kwargs) -> None:
         data.pop('releases', None)
 
         # break out the contents of the 'info' dict
-        data.update(data.pop('info'), {})
+        for k, v in data.pop('info', {}).items():
+            data[k] = v
```
Comment (bafulton) [inline r2c_isg/apis/pypi.py]: This is a bugfix--not sure why pypi was failing on the original code.

File: r2c_isg/cli.py [hunk 4]
```diff
@@ -55,7 +55,7 @@ def cli(ctx, debug, quiet):
               'days, sort by descending number of downloads, trim to the top '
               '100 most downloaded, download project metadata and all '
               'versions, and generate an input set json.\n'
-              '    load pypi top5kyear\n    sort "desc download_count"\n'
+              '    load pypi top4kyear\n    sort "desc download_count"\n'
```
Comment (bafulton) [inline r2c_isg/cli.py]: Apparently they're only adding the top 4k now. Go figure.

Comment (minusworld) [reply r2c_isg/cli.py]: 🤦‍♂️ 😆

File: r2c_isg/loaders/web/npm_loader.py [hunk 5]
```diff
@@ -34,21 +34,26 @@ def weblists(cls) -> dict:
         }
 
     @classmethod
-    def load(cls, weblist: str, **kwargs) -> Dataset:
+    def load(cls, name: str, **kwargs) -> Dataset:
+        # get the request type (weblist vs. organization)
+        from_type = kwargs.pop('from_type')
+        if from_type == 'org':
+            raise Exception('NPM does not support loading package lists from org names.')
```
Comment (minusworld) [inline r2c_isg/loaders/web/npm_loader.py]: Nice. Thank you for informative feedback loops :D

**Comment by ievans** [comment]
@bafulton @minusworld could we add an example to the README?

File: r2c_isg/loaders/web/github_loader.py [hunk 6]
```diff
@@ -19,24 +19,56 @@ def weblists(cls) -> dict:
...
+            # parse the data
+            GithubLoader._parse_github(ds, data)
 
         return ds
 
+    @staticmethod
+    def _get_org_repo_list(api, org_name, **kwargs):
+        # load the (paginated) list of repos for this organization
+        all_data = []
+        url = 'https://api.github.com/users/%s/repos?page=' % org_name
```
Comment (mschwager) [inline r2c_isg/loaders/web/github_loader.py]: We may want to create separate methods and CLI options for gathering a user's repos vs. an org's repos. I believe they're separate things in Github: [
List user repositories](https://developer.github.com/v3/repos/#list-user-repositories) vs. [
List organization repositories](https://developer.github.com/v3/repos/#list-organization-repositories).

**Comment by DrewDennison** [comment]
ship it! @bafulton

File: r2c_isg/loaders/web/github_loader.py (hunk 6 above)
Comment (bafulton) [reply r2c_isg/loaders/web/github_loader.py]: Whoops, I just committed a fix to master instead of proposing a PR. So, uh, I guess you've got your orgs vs. users support now! https://github.com/returntocorp/inputset-generator/commit/343ad475807a09cc6625a446c31e714d9c1e1763
